Dockerfile

```diff
@@ -1,12 +1,12 @@
 ############# builder
-FROM golang:1.14.9 AS builder
+FROM golang:1.15.3 AS builder
 
 WORKDIR /go/src/github.com/gardener/gardener-extension-provider-aws
 COPY . .
 RUN make install
 
 ############# base
-FROM alpine:3.12.0 AS base
+FROM alpine:3.12.1 AS base
```

Comment:
Can you also adapt
https://github.com/gardener/gardener-extension-provider-aws/blob/111ec9999369ae29f2354105b374561895e6c976/charts/gardener-extension-provider-aws/templates/_images.tpl#L2
(image of `mtu-customizer`)?